app/grandchallenge/cases/models.py

```diff
@@ -503,6 +503,12 @@ def update_viewer_groups_permissions(self, *, exclude_jobs=None):
         for g in groups_with_extra_perms:
             remove_perm("view_image", g, self)
 
+    def update_answer_image_viewer_permissions(self):
+        for answer in self.answer_set.all():
+            assign_perm("view_image", answer.creator, self)
+            editors = answer.question.reader_study.editors_group
+            assign_perm("view_image", editors, self)
```

Comment:
I think that there is an issue with this - if we add an image from an answer as an input to an algorithm (we don't do that right now but I can see it happening), then the editors permission will be removed by L504.